arc

```diff
@@ -132,47 +152,44 @@ fi
 debug "ARC_MLIR: $ARC_MLIR"
 
 if [ -z "$ARC_RUNTIME_DIR" ]; then
-    ARC_RUNTIME_DIR="$ROOT/arc-runtime/"
+    export ARC_RUNTIME_DIR="$ROOT/arc-runtime/"
 fi
 
 debug "ARC_RUNTIME_DIR: $ARC_RUNTIME_DIR"
 
 mkdir -p "$CRATE_DIR/src"
 
 # Create the Cargo.toml
```

Comment:
When this PR has been merged, should we break out the toml-template so it could be shared between this driver and the `arc-mlir-rust-test.in` script?

Reply:
Yes 👍 I think we could also remove `serde` and `hexf` from the dependencies list and re-export them from `arc-runtime`.